pytest.ini

```diff
@@ -0,0 +1,20 @@
+[pytest]
+testpaths = tests
+env =
+    SQLALCHEMY_DATABASE_URI=postgresql://postgres:postgres@localhost/test_notification_api
```

Comment:
this is `localhost` locally, but on jenkins it is `postgres`

Reply:
Hopefully this is fixed with `D:SQLALCHEMY_DATABASE_URI=postgresql://@localhost/test_notification_api`, which only sets the variable if it’s not already in the environment.